src/components/form-sections/PreviousEnrollmentSection/index.js

```diff
@@ -70,17 +71,6 @@ const PreviousEnrollmentSection = ({
     }
   }, [isUserEnrollmentLoading, disabled, previousTargetApi]);
 
-  const handleValidatedEnrollmentChange = event => {
-    const id = event.target.value;
-
-    onChange({
-      target: {
-        name: 'previous_enrollment_id',
-        value: id,
-      },
-    });
-  };
-
   return (
     <>
       {!disabled && !isUserEnrollmentLoading && (
```

Comment:
this is repeated 4x time, maybe we could set it in a more meaningful variable name

Reply:
we will delete this component in the future